src/vendor/vendor.c

```diff
@@ -1885,7 +1867,7 @@ int DiscoverDM_ForComponent(rdk_component_t *rdkc, kv_vector_t *rdk_objects, kv_
 ** \return  None
 **
 **************************************************************************/
-void Add_NameToDM(rdk_component_t *rdkc, char *instantiated_path, char *write_status, kv_vector_t *rdk_objects, kv_vector_t *rdk_params)
+void Add_NameToDM(const char *group, char *instantiated_path, char *write_status, kv_vector_t *rdk_objects, kv_vector_t *rdk_params)
 {
     char schema_path[MAX_DM_PATH];
     int len;
```

Comment:
I'm not sure if you need to call ConvertInstantiatedToSchemaPath() with the RBus APIs anymore. Does rbusElementInfo_get() return instantiated or supported data model ? ConvertInstantiatedToSchemaPath() is only needed if it's returning instantiated data model. If there is an RBus API function that returns supported data model, then that should be called instead. You want to get supported data model if you can get it.

Reply:
-> rbusElementInfo_get() returns entire data model(example: Device.X_RDK_Ethernet.Link.1.Enable) and its value
-> But ConvertInstantiatedToSchemaPath() is required due to below error
`DM_PRIV_AddSchemaPath: Device.X_RDK_Ethernet.Link.{i} must be registered before Device.X_RDK_Ethernet.Link.1.Enable`

Reply:
Yes, I looked into rbusElementInfo_get() and it's returning instantiated data model (it's basically CWMP's GetParameterNames RPC), so ConvertInstantiatedToSchemaPath() is required.